webapp/portlet/src/main/webapp/vue-apps/who-is-online-app/components/ExoWhoIsOnline.vue

```diff
@@ -42,10 +43,16 @@ export default {
     },
   },
   created() {
-    this.initOnlineUsers(this.$root.onlineUsers && this.$root.onlineUsers.users || []);
-    setInterval(function () {
-      this.retrieveOnlineUsers();
-    }.bind(this), this.delay);
+    const component = this;
+    document.onreadystatechange = function () {
+      if (document.readyState === 'complete' && !component.loaded) {
+        component.initOnlineUsers(component.$root.onlineUsers && component.$root.onlineUsers.users || []);
```

Comment:
We may have to use this function outside the event listener to initialize users `href` data correctly.
You can put here the code of `this.initPopup()` instead of `this.initOnlineUsers()`

**Note**: please make sure the following instruction at the end of  `initOnlineUsers` function :
```javascript
      return this.$nextTick()
        .then(() => this.$root.$applicationLoaded());
```

Reply:
done